.npmignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test
```

Comment:
@bengourley I just added .npmignore so `test` directory is not downloaded everytime people use the package.